script.js

```diff
@@ -0,0 +1,96 @@
+const API_URL = "https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=7d2a446aceb11d91875ece5f9a7f7d14&page=1";
+const IMG_PATH = "https://image.tmdb.org/t/p/w1280";
+const SEARCH_API = 'https://api.themoviedb.org/3/search/movie?api_key=7d2a446aceb11d91875ece5f9a7f7d14&query="'
+
+
+const form = document.getElementById("form");
+const searchMovie = document.getElementById("search");
+const main = document.getElementById("main");
+
+getMovies(API_URL);
+
+async function getMovies(url) {
+    try {
+        const response = await fetch(url);
+        const data = await response.json();
+
+        if (data.results) {
+            console.log(data.results);
+            showMovies(data.results);
+        } else {
+            console.log("No data");
+        }
+    } catch (error) {
+        console.error('Error:', error);
+    }
+}
+
+form.addEventListener("submit", (event) => {
+    event.preventDefault();
+
+    const searchKeyword = searchMovie.value;
+
+    if (searchKeyword && searchKeyword !== "") {
+        getMovies(SEARCH_API + searchKeyword) ;
+        searchMovie.value = "";
+    } else {
+        window.location.reload();
+    }
+})
+
+function showMovies(movies) {
+    main.innerHTML = "";
+
+    movies.forEach(movie => {
+        const {title, poster_path, overview, vote_average, release_date
+        } = movie;
+        
+        const movieElement = document.createElement("div");
+        movieElement.classList.add("movie");
+        movieElement.innerHTML = `
```

Comment:
Don't add code as innerHTML like this. Instead, you need to create all different HTML elements and append them to each other. Similar like we did in the lessons, e.g. here: https://github.com/frontend-from-0/group240123-files/blob/master/Lesson21/app.js

The innerHTML method is considered an insecure method and it's best to avoid it :) 
https://medium.com/@verity.carlos/why-you-shouldnt-use-innerhtml-and-what-to-use-instead-ed99d064a416

Reply:
This seems very complicated :) Wow, do we really need this?